irohad/consensus/sumeragi.cpp

```diff
@@ -143,72 +113,17 @@ namespace consensus {
      */
 
     int getNextOrder() {
-      thread_local int
-          currentProxyTail = static_cast<int>(getNumValidatingPeers()) - 1;
-      if (currentProxyTail >= peer_service::monitor::getActivePeerSize()) {
-        return -1;
-      }
-      return currentProxyTail++;
+      return 0;
```

Comment:
What does this mean? We also use 0th as leader?

If something is not finished add a TODO